taxi/views.py

```diff
@@ -34,6 +35,26 @@
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+    template_name = "taxi/manufacturer_form.html"
+
+
+class ManufacturerUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Manufacturer
+    fields = "__all__"
```

Comment:
Using `fields = "__all__"` in `UpdateView` exposes all fields of the `Manufacturer` model in the form. Ensure that this is intentional and that no sensitive fields are included. Consider specifying only the fields you want to expose.